jobs/Neatline_Job_DuplicateRecords.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @package     omeka
+ * @subpackage  neatline
+ * @copyright   2014 Rector and Board of Visitors, University of Virginia
+ * @license     http://www.apache.org/licenses/LICENSE-2.0.html
+ */
+
+
+class Neatline_Job_DuplicateRecords extends Omeka_Job_AbstractJob
```

Comment:
This looks like a good implementation; it raises a couple of possible questions though: 

- Does it seem clear that the intent is to clone all records, or to reuse existing ones, in a new exhibit?
- If we do clone all records, and there are several thousand (or million, as in [this example](https://www.youtube.com/watch?v=tPofW7j_6Y0)), a) this could blow things up, and b) we might want dependent delete upon exhibit deletion.
- If we change this so the clone does share records with the parent (which it currently doesn't, if I understand correctly), then we have to be sure we want to be sure we're **not** allowing dependent delete.